client/components/quizList/QuizListCard.jsx

```diff
@@ -1,30 +1,59 @@
 import React, { useState, useEffect } from 'react';
 import {
+  Box,
   Card,
   CardHeader,
   CardContent,
   Typography
 } from '@material-ui/core';
 import { useHistory } from 'react-router-dom';
+import { getFriends } from '../../../api_master';
 
-const  QuizListCard = ({ quiz }) => {
+import ChallengeFriend from '../friends/ChallengeFriend';
+
+const  QuizListCard = ({ quiz, loggedInUser }) => {
+
+  const [friends, setFriends] = useState([]);
+
+  const axiosRefreshFriends = () => {
+    getFriends(loggedInUser.id)
+      .then(friends => setFriends(friends.data.rows))
+      .catch(err => console.error(err))
+  }
 
   let history = useHistory();
+
   const handleClick = () => {
     history.push(`/quiz/${quiz.id}`);
   };
 
+  useEffect(() => {
+    axiosRefreshFriends();
+    return () => {
+      setFriends([]);
+    };
+  }, []);
```

Comment:
If you're doing a useEffect on the initial render, is the return statement necessary since the default value is already an empty array?

Reply:
No idea. I ripped that from Mike or Lukas.